src/RL/representation/default/heterogeneousstaterepresentation.jl

```diff
@@ -226,6 +234,16 @@ function initChosenFeatures!(sr::HeterogeneousStateRepresentation{DefaultFeaturi
             variable_counter += 1
         end
 
+        if haskey(chosen_features, "variable_is_objective") && chosen_features["variable_is_objective"]
+            sr.chosenFeatures["variable_is_objective"] = (true, variable_counter)
+            variable_counter += 1
+        end
+
+        if haskey(chosen_features, "nb_not_bounded_variable") && chosen_features["nb_not_bounded_variable"]
+            sr.chosenFeatures["nb_not_bounded_variable"] = (true, variable_counter)
+            variable_counter += 1
+        end
```

Comment:
@louis-gautier, this 4 lines are very close from line 246 to 249, I'm wondering if this is a copy/paste mistake ?

Reply:
It was.